src/components/ObjectInfo.vue

```diff
@@ -9,7 +9,12 @@
       <i class="fas fa-hashtag fa-fw"></i> ID
       <span v-if="withPermalink"><router-link :to="{ query: { id: `${data.map_type},${data.map_name},${data.hash_id}` } }" append>{{formatObjId(data.hash_id)}}</router-link></span>
       <span v-if="!withPermalink">{{formatObjId(data.hash_id)}}</span>
-      <span style="color: #ff3915" v-if="data.hard_mode"> Master Mode</span>
+    </section>
+    <section class="search-result-id">
+      <span style="color: #ff3915" v-if="data.hard_mode"><i class="fa fa-shield-alt fa-fw"></i> Master Mode</span>
+    </section>
+    <section class="search-result-id">
+      <span style="color: #ff3915" v-if="data.one_hit_mode"><i class="fa fa-shield-alt fa-fw"></i> One-hit Obliterator Object</span>
```

Comment:
```suggestion
      <span style="color: #ff3915" v-if="data.one_hit_mode"><i class="fa fa-shield-alt fa-fw"></i> One-Hit Obliterator Object</span>
```

Also can we use different icons for the Master Mode and OhO indicators? (fa-shield-alt is already used for enemy equipment)